public/assets/css/article.css

```diff
@@ -1,38 +1,88 @@
 article {
```

Comment:
in your place, I would have created several files in an article folder.

`css/article/article.css` , `css/article/header.css` ,  `css/article/footer.css`,   `css/article/content.css` 
and  `css/article/article.css`  would contain a global rule of the articles and would import the other files.